comment/utils.py

```diff
@@ -79,12 +79,20 @@ def has_valid_profile():
 
 
 def is_comment_admin(user):
-    return user.groups.filter(name='comment_admin').exists() or \
-           (user.has_perm('comment.delete_flagged_comment') and user.has_perm('comment.delete_comment'))
+    if settings.COMMENT_ALLOW_MODERATION:
```

Comment:
```suggestion
    if settings.COMMENT_FLAGS_ALLOWED`:
```

Reply:
I think there's an extra " ` " at the end there, no?